README.md

```diff
@@ -98,15 +98,17 @@ Thanks! And great!
 
 This work is still in early development, which means it's _not ready for production_, but also that your feedback can have a big impact.
 
-You can reach us here, in this repository via issues and discussion, or:
+You can reach us here, in this repository via [issues](https://github.com/kcp-dev/kcp/issues) and [discussions](https://github.com/kcp-dev/kcp/discussions), or:
 
 - Join the [`#kcp-prototype` channel](https://app.slack.com/client/T09NY5SBT/C021U8WSAFK) in the [Kubernetes Slack workspace](https://slack.k8s.io)
 - Join the mailing lists
     - [kcp-dev](https://groups.google.com/g/kcp-dev) for development discussions
     - [kcp-users](https://groups.google.com/g/kcp-users) for discussions among users and potential users
 - Subscribe to the [community calendar](https://calendar.google.com/calendar/embed?src=ujjomvk4fa9fgdaem32afgl7g0%40group.calendar.google.com) for community meetings and events
     - The kcp-dev mailing list is subscribed to this calendar
-- Browse the [shared Google Drive](https://drive.google.com/drive/folders/1FN7AZ_Q1CQor6eK0gpuKwdGFNwYI517M?usp=sharing) to share design docs, demo videos, meeting recordings
+- See recordings of past community meetings on [YouTube](https://www.youtube.com/channel/UCfP_yS5uYix0ppSbm2ltS5Q)
+- See [upcoming](https://github.com/kcp-dev/kcp/issues?q=is%3Aissue+is%3Aopen+label%3Acommunity-meeting) and [past](https://github.com/kcp-dev/kcp/issues?q=is%3Aissue+label%3Acommunity-meeting+is%3Aclosed) community meeting agendas and notes
+- Browse the [shared Google Drive](https://drive.google.com/drive/folders/1FN7AZ_Q1CQor6eK0gpuKwdGFNwYI517M?usp=sharing) to share design docs, notes, etc.
```

Comment:
@imjasonh just want to confirm with you, this google drive is still empty right now? As I cannot see anything from this folder.

Reply:
I see one item, another copy of the recording of the community meeting. It's shared with kcp-dev@ as commenters.

I probably won't put future recordings in there too, but the drive should still be visible/usable for sharing design docs, etc.

Reply:
@imjasonh thanks. After join the kcp-dev group, I can see the recording. But one question: Is it possible to enable it can public to everyone even if they do not join the kcp-dev group? Or else we have to highlight join the `kcp-dev` group is a must if want to view the google drive documents.